src/main/java/com/bootcamp/demo/business/AccountService.java

```diff
@@ -30,45 +29,60 @@ public void setEncryptor(Encryptor encryptor) {
         this.encryptor = encryptor;
     }
 
-    public boolean accountValidation(Account account){
+    public String accountValidation(Account account) throws ServiceException {
+
+        StringBuilder stringBuilder = new StringBuilder();
         if(account.getName().isEmpty() || account.getPassword().isEmpty() || account.getEmail().isEmpty() || account.getUsername().isEmpty() || account.getAddress().isEmpty() || account.getPhoneNumber().isEmpty() || account.getDateOfBirth().isEmpty())
```

Comment:
Is it good enough to have a generic message for either field is missing or should we have specific ones